server/src/start-worker.ts

```diff
@@ -8,6 +8,7 @@ export async function startWorker() {
     const config = loadAppConfigFromEnv(process.env);
     const app = await bootstrap(config);
     runDistributionWorker(app, config.distributionInterval * MILLISECONDS_PER_MINUTE);
+    runVettedBeneficiaryDistributionWorker(app, config.distributionInterval * MILLISECONDS_PER_MINUTE);
```

Comment:
Let's use a separate interval variable (and env variable) for vetted distribution: `config.vettedDistributionInterval`, because we want these to run at different periods.